File: 64Bit/extensions/workbook/ws_stbar_menu.lua

local MDL_OPNBROWS = "extensions."..ACTIVEDIR_STEM..".urls.openbrowser"

local btnSciSuit = std.Button.new{
	title = "PE Bytes", 
	module = MDL_OPNBROWS, 
	call = "Open",
	param = "https://www.pebytes.com",
	img = ACTIVEDIR.."/icons/pebytes.png"}


local btnPython = std.Button.new{
	title = "Python", 
	module = MDL_OPNBROWS, 
	call = "Open",
	param = "https://www.python.org/",
	img = ACTIVEDIR.."/icons/py_logo32.png"}

local btnNumpy = std.Button.new{
	title = "Numpy", 
	module = MDL_OPNBROWS, 
	call = "Open",
	param = "https://www.numpy.org",
	img = ACTIVEDIR.."/icons/numpy.png"}

local btnPandas = std.Button.new{
	title = "Pandas", 
	module = MDL_OPNBROWS, 
	call = "Open",
	param = "https://pandas.pydata.org/",
	img = ACTIVEDIR.."/icons/pandas.png"}



std.append(btnSciSuit, 1) -- append to 1st rectangle field
 --std.append(3) --separator

std.append(btnPython, 3) -- append to 3rd rectangle field
std.append(btnNumpy, 3)
std.append(btnPandas, 3)

File: 64Bit/extensions/workbook/ws_stbar_menu.py
import pathlib

import _sci.extension as ext
from _sci.framework import Framework

import wx

def Open(URL):
	wx.LaunchDefaultBrowser(URL)


if __name__ == "__main__":
	CurFile = str(__file__)
	CurFolder = pathlib.Path(__file__).parent

	btnSciSuit = ext.Button("PE Bytes", str(CurFolder/"icons/pebytes.png"), CurFile, Open, "https://www.pebytes.com")
	btnPython = ext.Button("Python", str(CurFolder/"icons/py_logo32.png"), CurFile, Open, "https://www.python.org/")
	btnNumpy = ext.Button("Numpy", str(CurFolder/"icons/numpy.png"), CurFile, Open, "https://www.numpy.org")
	btnPandas = ext.Button("Pandas", str(CurFolder/"icons/pandas.png"), CurFile, Open, "https://pandas.pydata.org/")
	
	
	Framework().StatusBar().AppendMenuItem(1, btnSciSuit) 

	Framework().StatusBar().AppendMenuItem(3, btnPython) 
	Framework().StatusBar().AppendMenuItem(3, btnNumpy)
	Framework().StatusBar().AppendMenuItem(3, btnPandas)
